packages/cli/src/commands/sync.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import { createHmac } from 'crypto';
+
+export async function sync(bridgeUrl: string, novuApiKey: string, novuCloudApiUrl: string) {
```

Comment:
Please throw an error if the secretKey is missing suggesting resolution to the developer. See [here](https://github.com/novuhq/novu/pull/5857)

Reply:
Renames per DX guide

```suggestion
export async function sync(endpointUrl: string, secretKey: string, apiUrl: string) {
```